pulp_ansible/tests/functional/api/collection/test_sync.py

```diff
@@ -6,7 +6,7 @@
 )
 from pulp_ansible.tests.functional.constants import TEST_COLLECTION_CONFIGS
 from orionutils.generator import build_collection
-from pulpcore.client.pulp_ansible import PulpAnsibleGalaxyApiV3CollectionsApi
+from pulpcore.client.pulp_ansible import PulpAnsibleApiV3ArtifactsCollectionsApi
```

Comment:
Is it possible to remove the additional `Api` on the end of all the bindings names here? So `PulpAnsibleApiV3ArtifactsCollectionsApi` would become `PulpAnsibleApiV3ArtifactsCollections`?

Reply:
we can by setting `pulp_tag_name` https://docs.pulpproject.org/pulpcore/plugins/reference/how-to-doc-api.html#openapi-tags

Reply:
actually, the "Api" at the end is appended by https://github.com/pulp/pulp-openapi-generator